Update news (#7729)
microsoft/AL: News.md

# 2024-04-24 Transient failures while running v.24 updates and publishing apps from Visual Studio Code
 

> We have identified an issue where publishing apps may sometime fail due to a race condition in the flow of installing or updating multiple apps, causing certain development operations like app publishing and updates to v.24 to fail with internal error. For troubleshooting update issues, please see the Operations Log page in the Business Central Admin Center to find the details of the errors discovered during update or app publishing.

Reviewer: ```suggestion
We have identified an issue where publishing apps might sometimes fail due to a race condition in the flow of installing or updating multiple apps, causing certain development operations like app publishing and updates to v. 24 to fail with an internal error. For troubleshooting update issues, see the **Operations Log** page in the Business Central Admin Center. Here you'll find the details of the errors discovered during update or app publishing.
```

 
We are actively working on identifying the root cause and mitigating the issue.
 

> While we are working on addressing this instability, the issue can be mitigated by re-trying the operation, e.g. re-trying app publishing or update, until it succeeds. When deploying PTEs as DEV extensions from Visual Studio code fails, another option is to upload it via the Extension Management page in the web client, and optionally subsequently use debug without publishing if performing debugging.

Reviewer: ```suggestion
While we are working on addressing this instability, the issue can be mitigated by re-trying the operation, for example, re-trying app publishing or update, until it succeeds. When deploying PTEs as DEV extensions from Visual Studio code fails, another option is to upload it through the **Extension Management** page in the web client, and optionally subsequently use debug without publishing if you're doing debugging.
```

> We will delete this entry when the issue has been resolved. In the meantime, you can follow or provide details on the GitHub AL issue on [deploying through Visual Studio code fails](https://github.com/microsoft/AL/issues/7721)
>  
> Comms on this have been posted under ID DN78455 if you need to refer to that in any related support case.

Reviewer 2: ```suggestion
**Please do not open any new, duplicate support cases on above**. Comms on this have been posted under ID DN78455 if you need to refer to that in any existing, related support case.
 
We will delete this entry when the issue has been resolved. 

In the meantime, you can follow or provide details on the GitHub AL issue on [deploying through Visual Studio code fails](https://github.com/microsoft/AL/issues/7721)
```

> Comms on this have been posted under ID DN78455 if you need to refer to that in any related support case.

Reviewer: ```suggestion
Communication regarding this issue are posted under ID DN78455 in case you need to refer to that in any related support case.
```

# 2024-04-24 Transient failures while running v.24 updates and publishing apps from Visual Studio Code
 
We have identified an issue where publishing apps may sometime fail due to a race condition in the flow of installing or updating multiple apps, causing certain development operations like app publishing and updates to v.24 to fail with internal error. For troubleshooting update issues, please see the Operations Log page in the Business Central Admin Center to find the details of the errors discovered during update or app publishing.
 
We are actively working on identifying the root cause and mitigating the issue.
 
While we are working on addressing this instability, the issue can be mitigated by re-trying the operation, e.g. re-trying app publishing or update, until it succeeds. When deploying PTEs as DEV extensions from Visual Studio code fails, another option is to upload it via the Extension Management page in the web client, and optionally subsequently use debug without publishing if performing debugging.
 

> **Please do not open any new, duplicate support cases on above**. Comms on this have been posted under ID DN78455 if you need to refer to that in any existing, related support case.

Reviewer: Suggestion:

Communication on this issue are posted under...
